custom_components/pyscript/state.py

```diff
@@ -115,12 +115,12 @@ async def update(cls, new_vars, func_args):
     @classmethod
     def notify_var_get(cls, var_names, new_vars):
         """Return the most recent value of a state variable change."""
-        notify_vars = {}
+        notify_vars = new_vars
```

Comment:
This looks like a good simplification.  One small change: it's probably better to do
```
notify_vars = new_vars.copy()
```
so that `new_vars` isn't modified in-place.

Reply:
I honestly don't know why this part wasn't working the way it was. I made this change just to be SURE, and it ended up fixing it.

Reply:
I honestly don't know why this wasn't working the way it was. I made this change just to be SURE it was what I expected it to be, and it fixed the whole thing. I guess cls.notify had something in it I didn't expect. But I'm not sure how.